public/stylesheets/style.css

```diff
@@ -0,0 +1,24 @@
+
+* {
+  border-radius: 0 !important;
+}
+
+.splash-background {
+  background: url(../home_splash.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
I would like to avoid the use of manually defined, browser specific style properties.  We need to capture an issue to address this, as well as to define our style approach (I'm thinking something along the line of BEM currently), but please remove these for now (which may necessitate removal of the image).